nttt/tidyup.py

```diff
@@ -71,7 +100,7 @@ def tidyup_translations(arguments):
 
                 print("Fixing - {}".format(file_name))
                 if os.path.basename(source_file_path) == "meta.yml":
-                    fix_meta(source_file_path, output_file_path)
+                    fix_meta(source_file_path, english_folder, output_file_path)
```

Comment:
Function fix_meta expects a file path as second argument (like the other arguments), but here a directory is passed.